src/python/doc/alpha_complex_user.rst

```diff
@@ -89,25 +89,28 @@ In order to build the alpha complex, first, a Simplex tree is built from the cel
 Filtration value computation algorithm
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-  **for** i : dimension :math:`\rightarrow` 0 **do**
-    **for all** :math:`\sigma` of dimension i
-      **if** filtration(:math:`\sigma`) is NaN **then**
-        filtration(:math:`\sigma`) = :math:`\alpha^2(\sigma)`
-      **end if**
+.. code-block:: bash
+
+    for i : dimension → 0 do
+      for all σ of dimension i
+        if filtration(σ) is NaN then
+          filtration(σ)=α2(σ)
```

Comment:
```suggestion
          filtration(σ) = α²(σ)
```
?

Reply:
Bad copy / paste. I fixed it.

Reply:
I was also suggesting some spaces around `=` to be consistent with the rest of the code in the block.